app/build.gradle

```diff
@@ -60,6 +60,10 @@ dependencies {
     //Timber
     implementation "com.jakewharton.timber:timber:$timber"
 
+    //Navigation
+    implementation "android.arch.navigation:navigation-fragment-ktx:$navigation"
```

Comment:
are we gonna use navigation, shouldn't we discuss it first? the benefits and problems?